### PR TITLE
update github ci workflow fix e2e run failed

### PR DESCRIPTION
**Pull request by village-way**
#### What type of PR is this?
/kind failing-test

#### Which issue(s) does this PR fix?

Fixes #
#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

File: hack/rune2e.sh [hunk 1]
```diff
@@ -6,6 +6,7 @@ set -o pipefail
 
 KUBECONFIG_PATH=${KUBECONFIG_PATH:-"${HOME}/.kube"}
 export KUBECONFIG=$KUBECONFIG_PATH/"config"
+echo "$KUBECONFIG"
```
Comment (duanmengkk) [inline hack/rune2e.sh]: delete unsed code

File: hack/rune2e.sh [hunk 2]
```diff
@@ -15,6 +16,7 @@ MEMBER3_CLUSTER_NAME="cluster-member3"
 
 ROOT="$(dirname "${BASH_SOURCE[0]}")"
 source "${ROOT}/util.sh"
+echo "$ROOT"
```
Comment (duanmengkk) [inline hack/rune2e.sh]: echo "$KUBECONFIG"

File: hack/verify-vendor.sh [hunk 3]
```diff
@@ -4,6 +4,8 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+git init
```
Comment (duanmengkk) [inline hack/verify-vendor.sh]: Is this really need？

File: hack/install-golint.sh [hunk 4]
```diff
@@ -0,0 +1,419 @@
...
+log_info "found version: ${VERSION} for ${TAG}/${OS}/${ARCH}"
+
+NAME=${BINARY}-${VERSION}-${OS}-${ARCH}
+TARBALL=${NAME}.${FORMAT}
+TARBALL_URL=${GITHUB_DOWNLOAD}/${TAG}/${TARBALL}
+CHECKSUM=${PROJECT_NAME}-${VERSION}-checksums.txt
+CHECKSUM_URL=${GITHUB_DOWNLOAD}/${TAG}/${CHECKSUM}
+
+
+execute
```
Comment (duanmengkk) [inline hack/install-golint.sh]: Why is this piece of code so long

Comment (village-way) [reply hack/install-golint.sh]: the code is from golint project. updated by deleted and use curl to get install script

File: hack/rune2e.sh (hunk 2 above)
Comment (village-way) [reply hack/rune2e.sh]: fixed by delete code `echo "$ROOT"`

File: hack/rune2e.sh (hunk 1 above)
Comment (village-way) [reply hack/rune2e.sh]: fixed by delete code `echo "$KUBECONFIG"`

**Pull request merged**